exampleSite/content/search/index.md

```diff
@@ -0,0 +1,6 @@
++++
```

Comment:
If the example is meant to be used to showcase for the user later too, we should start with using `fragments/search` right away.